### ISSUE TITLE
[Bug]: test_delete_rbd_pool_attached_to_sc_UI fails due to TimeoutException: Message: Failed to find the element (xpath,//tr[contains(., 'test-pool')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'test-pool')]//button[@data-test='kebab-button'])

### ISSUE THREAD

**Issue by sidhant-agrawal** [red-hat-storage/ocs-ci#13707]
### Description
test_delete_rbd_pool_attached_to_sc_UI fails due to TimeoutException
```
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (xpath,//tr[contains(., 'test-pool')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'test-pool')]//button[@data-test='kebab-button'])
```

Output from console logs
```
 05:40:03 - MainThread - ocs_ci.ocs.ui.base_ui - INFO  - Deleting the block pool: test-pool
 05:40:03 - MainThread - ocs_ci.ocs.ui.base_ui - INFO  - Search-by option is already 'name'
 05:40:16 - MainThread - ocs_ci.ocs.ui.base_ui - INFO  - page loaded: https://console-openshift-console.apps.sagrawal-gsq.qe.rh-ocs.com/odf/storage-cluster/storage-pools?name=test-pool
 05:40:52 - MainThread - ocs_ci.ocs.ui.base_ui - INFO  - Copy DOM file: /home/jenkins/current-cluster-dir/logs/ui_logs_dir_1763548607990/dom/test_delete_rbd_pool_attached_to_sc_UI[3-aggressive-Immediate-Bound]/2025-11-19T05-40-52.877413_StoragePools-2025-11-19T05-40-16.374486_DOM.txt
 05:40:53 - MainThread - ocs_ci.ocs.ui.base_ui - ERROR  - Message: 
 Stacktrace:
 #0 0x562ac5d4e71a <unknown>
 #1 0x562ac57f10a0 <unknown>
 #2 0x562ac58429b0 <unknown>
 #3 0x562ac5842ba1 <unknown>
 #4 0x562ac5890ea4 <unknown>
 #5 0x562ac58683cd <unknown>
 #6 0x562ac588e2a0 <unknown>
 #7 0x562ac5868173 <unknown>
 #8 0x562ac5834d4b <unknown>
 #9 0x562ac58359b1 <unknown>
 #10 0x562ac5d138cb <unknown>
 #11 0x562ac5d177ca <unknown>
 #12 0x562ac5cfb622 <unknown>
 #13 0x562ac5d18354 <unknown>
 #14 0x562ac5ce045f <unknown>
 #15 0x562ac5d3c4f8 <unknown>
 #16 0x562ac5d3c6d6 <unknown>
 #17 0x562ac5d4d586 <unknown>
 #18 0x7f827028b2fa start_thread
 
 05:40:53 - MainThread - ocs_ci.framework.pytest_customization.reports - INFO  - duration reported by tests/functional/storageclass/test_delete_rbd_pool_attached_to_sc.py::TestDeleteRbdPool::test_delete_rbd_pool_attached_to_sc_UI[3-aggressive-Immediate-Bound] immediately after test execution: 118.93
 FAILED
 _ TestDeleteRbdPool.test_delete_rbd_pool_attached_to_sc_UI[3-aggressive-Immediate-Bound] _
 
 _locator = ("//tr[contains(., 'test-pool')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'test-pool')]//button[@data-test='kebab-button']", 'xpath')
 _timeout = 30, _enable_screenshot = True, _copy_dom = False
 
     def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
         # wait for page fully loaded only if an element was not located
         # prevents needless waiting and frequent crushes on ODF Overview page,
         # when metrics and alerts frequently updated
         if not self.get_elements(_locator):
             self.page_has_loaded()
         screenshot = (
             ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
         )
         date_time = datetime.datetime.now().strftime("%Y-%m-%dT%H-%M-%S.%f")
         if screenshot:
             self.take_screenshot(f"{type(self).__name__}-{date_time}")
         if _copy_dom:
             self.copy_dom(f"{type(self).__name__}-{date_time}")
     
         wait = WebDriverWait(self.driver, timeout)
         try:
 >           element = wait.until(
                 ec.element_to_be_clickable((locator[1], locator[0]))
             )
 
 ocs_ci/ocs/ui/base_ui.py:221: 
 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
 
 self = <selenium.webdriver.support.wait.WebDriverWait (session="e65caca28d7aeee63ea2dbad9fcf2fde")>
 method = <function element_to_be_clickable.<locals>._predicate at 0x7f53f8269260>
 message = ''
 
     def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = "") -> T:
         """Wait until the method returns a value that is not False.
     
         Calls the method provided with the driver as an argument until the
         return value does not evaluate to ``False``.
     
         Parameters:
         -----------
         method: callable(WebDriver)
             - A callable object that takes a WebDriver instance as an argument.
     
         message: str
             - Optional message for :exc:`TimeoutException`
     
         Return:
         -------
         object: T
             - The result of the last call to `method`
     
         Raises:
         -------
         TimeoutException
             - If 'method' does not return a truthy value within the WebDriverWait
             object's timeout
     
         Example:
         --------
         >>> from selenium.webdriver.common.by import By
         >>> from selenium.webdriver.support.ui import WebDriverWait
         >>> from selenium.webdriver.support import expected_conditions as EC
     
         # Wait until an element is visible on the page
         >>> wait = WebDriverWait(driver, 10)
         >>> element = wait.until(EC.visibility_of_element_located((By.ID, "exampleId")))
         >>> print(element.text)
         """
         screen = None
         stacktrace = None
     
         end_time = time.monotonic() + self._timeout
         while True:
             try:
                 value = method(self._driver)
                 if value:
                     return value
             except self._ignored_exceptions as exc:
                 screen = getattr(exc, "screen", None)
                 stacktrace = getattr(exc, "stacktrace", None)
             if time.monotonic() > end_time:
                 break
             time.sleep(self._poll)
 >       raise TimeoutException(message, screen, stacktrace)
 E       selenium.common.exceptions.TimeoutException: Message: 
 E       Stacktrace:
 E       #0 0x562ac5d4e71a <unknown>
 E       #1 0x562ac57f10a0 <unknown>
 E       #2 0x562ac58429b0 <unknown>
 E       #3 0x562ac5842ba1 <unknown>
 E       #4 0x562ac5890ea4 <unknown>
 E       #5 0x562ac58683cd <unknown>
 E       #6 0x562ac588e2a0 <unknown>
 E       #7 0x562ac5868173 <unknown>
 E       #8 0x562ac5834d4b <unknown>
 E       #9 0x562ac58359b1 <unknown>
 E       #10 0x562ac5d138cb <unknown>
 E       #11 0x562ac5d177ca <unknown>
 E       #12 0x562ac5cfb622 <unknown>
 E       #13 0x562ac5d18354 <unknown>
 E       #14 0x562ac5ce045f <unknown>
 E       #15 0x562ac5d3c4f8 <unknown>
 E       #16 0x562ac5d3c6d6 <unknown>
 E       #17 0x562ac5d4d586 <unknown>
 E       #18 0x7f827028b2fa start_thread
 
 venv/lib64/python3.11/site-packages/selenium/webdriver/support/wait.py:138: TimeoutException
 
 During handling of the above exception, another exception occurred:
 
 self = <tests.functional.storageclass.test_delete_rbd_pool_attached_to_sc.TestDeleteRbdPool object at 0x7f53f9ef9550>
 replica = 3, compression = 'aggressive', volume_binding_mode = 'Immediate'
 pvc_status = 'Bound'
 storageclass_factory_class = <function storageclass_factory_fixture.<locals>.factory at 0x7f53df93f420>
 pvc_factory = <function pvc_factory_fixture.<locals>.factory at 0x7f53f924c180>
 pod_factory = <function pod_factory_fixture.<locals>.factory at 0x7f53f924c4a0>
 setup_ui = <selenium.webdriver.chrome.webdriver.WebDriver (session="e65caca28d7aeee63ea2dbad9fcf2fde")>
 
     @tier3
     @skipif_external_mode
     @skipif_ibm_cloud_managed
     @pytest.mark.parametrize(
         argnames=["replica", "compression", "volume_binding_mode", "pvc_status"],
         argvalues=[
             pytest.param(
                 *[
                     3,
                     "aggressive",
                     constants.IMMEDIATE_VOLUMEBINDINGMODE,
                     constants.STATUS_BOUND,
                 ],
                 marks=pytest.mark.polarion_id("OCS-5151"),
             )
         ],
     )
     def test_delete_rbd_pool_attached_to_sc_UI(
         self,
         replica,
         compression,
         volume_binding_mode,
         pvc_status,
         storageclass_factory_class,
         pvc_factory,
         pod_factory,
         setup_ui,
     ):
         cbp_name = preconditions_rbd_pool_created_associated_to_sc(
             compression,
             pod_factory,
             pvc_factory,
             pvc_status,
             replica,
             storageclass_factory_class,
             volume_binding_mode,
         )
         blocking_pool_tab = PageNavigator().navigate_storage_pools_page()
 >       assert not blocking_pool_tab.delete_block_pool(
             cbp_name, cannot_be_deleted=True
         ), "blocking pool attached by storage class was deleted, no Warning message was shown"
 
 tests/functional/storageclass/test_delete_rbd_pool_attached_to_sc.py:237: 
 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
 ocs_ci/ocs/ui/page_objects/block_pools.py:117: in delete_block_pool
     self.do_click(resource_actions, enable_screenshot=True)
 ocs_ci/ocs/ui/base_ui.py:234: in do_click
     _do_click(locator, timeout, enable_screenshot, copy_dom)
 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
 
 _locator = ("//tr[contains(., 'test-pool')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'test-pool')]//button[@data-test='kebab-button']", 'xpath')
 _timeout = 30, _enable_screenshot = True, _copy_dom = False
 
     def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
         # wait for page fully loaded only if an element was not located
         # prevents needless waiting and frequent crushes on ODF Overview page,
         # when metrics and alerts frequently updated
         if not self.get_elements(_locator):
             self.page_has_loaded()
         screenshot = (
             ocsci_config.UI_SELENIUM.get("screenshot") and enable_screenshot
         )
         date_time = datetime.datetime.now().strftime("%Y-%m-%dT%H-%M-%S.%f")
         if screenshot:
             self.take_screenshot(f"{type(self).__name__}-{date_time}")
         if _copy_dom:
             self.copy_dom(f"{type(self).__name__}-{date_time}")
     
         wait = WebDriverWait(self.driver, timeout)
         try:
             element = wait.until(
                 ec.element_to_be_clickable((locator[1], locator[0]))
             )
             element.click()
         except TimeoutException as e:
             self.take_screenshot(f"{type(self).__name__}-{date_time}")
             self.copy_dom(f"{type(self).__name__}-{date_time}")
             logger.error(e)
 >           raise TimeoutException(
                 f"Failed to find the element ({locator[1]},{locator[0]})"
             )
 E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (xpath,//tr[contains(., 'test-pool')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'test-pool')]//button[@data-test='kebab-button'])
 
 ocs_ci/ocs/ui/base_ui.py:229: TimeoutException

```
### Steps to Reproduce
1. Run test tests/functional/storageclass/test_delete_rbd_pool_attached_to_sc.py::TestDeleteRbdPool::test_delete_rbd_pool_attached_to_sc_UI[3-aggressive-Immediate-Bound]

### Screenshots
https://url.corp.redhat.com/bcbdb68

### Additional Info
The test fails even if different name is used in the test.
For eg. 
```
E           selenium.common.exceptions.TimeoutException: Message: Failed to find the element (xpath,//tr[contains(., 'custom-poolname')]//button[@aria-label='Kebab toggle'] | //tr[contains(., 'custom-poolname')]//button[@data-test='kebab-button'])

ocs_ci/ocs/ui/base_ui.py:229: TimeoutException
```
